amqp_influxdb/__main__.py

```diff
@@ -47,10 +55,22 @@ def main():
         batch_size=args.influx_batch_size,
         max_batch_delay=args.influx_max_batch_delay)
 
+    # This arg is a string for ease of use with the manager blueprint
+    if args.amqp_ssl_enabled.lower() == 'true':
```

Comment:
This probably needs an additional check to make sure the cert path is present when it tries to run.

What's best for us to raise at this point, and/or should it be caught in the manager blueprint?